src/net/download.rs

```diff
@@ -118,6 +118,12 @@ pub const BLOCK_REREQUEST_INTERVAL: u64 = 30;
 /// This module is responsible for downloading blocks and microblocks from other peers, using block
 /// inventory state (see src/net/inv.rs)
 
+#[derive(Debug, PartialEq, Clone, Hash, Eq)]
+pub enum BlockRequestKeyKind {
+    Block,
+    ConfirmedMicroblock,
```

Comment:
Please rename to either `ConfirmedMicroblockStream` or `ConfirmedMicroblocks`.  The code never requests a single microblock.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/91807572bf5768b803b30df79b32ffeb6c41ee2a